torch/_dynamo/eval_frame.py

```diff
@@ -596,9 +602,53 @@ class directly; instead, use :func:`torch._export.dynamic_dim`.
     # TODO: We don't need t_id; we can get it off of w_tensor
     t_id: int
     dim: int
-    constraint_range: Optional[
-        torch.fx.experimental.symbolic_shapes.StrictMinMaxConstraint
-    ]
+    constraint_range: Optional[StrictMinMaxConstraint]
```

Comment:
I think you can substantially simplify the logic below if you make this non-Optional

Reply:
lol you're right, I conservatively, and mistakenly, assumed that `constraint_range=None` actually has a user-facing need, but it doesn't seem so. Killed it, so much simpler.